### PR TITLE
August 2015 bsides

### PR DESCRIPTION
**Pull request by GeordieMatt**
I had to go slightly off-piste for this one, because there were multiple mash steps in this batch. Let me know what you think! Also, Orange Blossom marked its last mash step as mashout, but I wasn't sure of the best way to mark that out (or if I needed it at all).

### CODE REVIEW

**Comment by sammdec** [comment]
Thats an interesting one! I will have a think, I wonder if it is worth making all the mash steps an array of objects

**Comment by sammdec** [comment]
OK so I pushed an update that fixes this issue, all you need to do is rebase this branch with master and the tests should pass fine

File: data/baby_saison.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
+{
+  "name": "Baby Saison",
```
Comment (sammdec) [inline data/baby_saison.json]: Could you add the B Sides prefix in at the end maybe e.g "Baby Saison - B-Sides"

File: data/morags_mojito.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
+{
+  "name": "Morag's Mojito",
```
Comment (sammdec) [inline data/morags_mojito.json]: Same thing here with the B-Sides thing

File: data/orange_blossom.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
+{
+  "name": "Orange Blossom",
```
Comment (sammdec) [inline data/orange_blossom.json]: Naming thing as before

**Comment by sammdec** [comment]
Apart from this couple of points and the rebase thing it all looks great :+1: thanks for contributing! :smile:

**Comment by GeordieMatt** [comment]
Just rebased and added those suffixes in. There might need to be a review of 'series' beers like Unleash the Yeast and IPA is Dead to make sure they're all marked the same. Happy to be on board!

**Comment by sammdec** [comment]
Yeah makes sense, I will make sure to check them, I dont think any have come through yet

**Pull request merged**